### PR TITLE
Feature/implement calculate macros

### PR DESCRIPTION
**Pull request by kimberllyflorez**

### CODE REVIEW

File: lib/domain/use_cases/profile/calculate_calories_use_case.dart [hunk 1]
```diff
@@ -1,23 +1,37 @@
...
+    final gramsCarbs = _carbs(getInt, gramsProtein, gramsFat);
+
+    return CaloriesModel(
+        carbsCal: gramsCarbs,
+        fatCal: gramsFat,
+        proteinCal: gramsProtein,
+        totalCal: getInt);
+  }
+
+  int _totalCalorie() {
```
Comment (curregoz) [inline lib/domain/use_cases/profile/calculate_calories_use_case.dart:34]: Rename the methods with a name that says by itself what it does

File: lib/domain/use_cases/profile/calculate_calories_use_case.dart [hunk 2]
```diff
@@ -44,4 +58,20 @@ class CalculateCalorieImpl extends CalculateCalorieUseCase {
     get = eta * dataUser.activityLevel;
     return get.floor();
   }
+
+  int _protein() {
```
Comment (curregoz) [inline lib/domain/use_cases/profile/calculate_calories_use_case.dart:62]: Same in this method and the methods above

Comment (kimberllyflorez) [reply lib/domain/use_cases/profile/calculate_calories_use_case.dart]: done

File: lib/domain/use_cases/profile/calculate_calories_use_case.dart (hunk 1 above)
Comment (kimberllyflorez) [reply lib/domain/use_cases/profile/calculate_calories_use_case.dart]: done

File: lib/app/home/ui/widget/statistics_wd.dart [hunk 3]
```diff
@@ -3,9 +3,18 @@ import 'package:percent_indicator/percent_indicator.dart';
 import 'package:projects_study/design_system/app_theme/app_theme.dart';
 
 class Statistics extends StatelessWidget {
```
[the remark this answers is not in the document]
Comment (kimberllyflorez) [reply lib/app/home/ui/widget/statistics_wd.dart]: done

**Review by curregoz — APPROVED** [review]

**Pull request merged**